packages/common/api.d.ts

```diff
@@ -10,3 +10,20 @@ export function findNextTextualToken(
   tokenVector: IToken[],
   prevTokEndOffset: number
 ): IToken | null;
+
+/**
+ * Check if an xml attribute key is an xmlns key.

```

Comment:
As this is a pubic API, maybe give two example snippets? (default xmlns and xmlns with prefix)